biorxiv/pca_association_experiment/biorxiv_pca_plots.py

```diff
@@ -0,0 +1,639 @@
+
+# In[27]:
+
+
+(
+    pca_df[["pca1", "pca6", "category", "doi"]]
+    .drop_duplicates("doi")
+    .query("pca1 > 0 & pca6 > 3")
+    .sort_values("pca6", ascending=False)
+    .head(10)
+)
+
+
+# ## PCA2 vs PCA15
+
+# In[28]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_02_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_15_cossim_word_cloud.png'
+)
+
+
+# The word cloud on the right seems to contain the following concepts: facial recognition and behavior vs neuron biochemistry.
+
+# In[29]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca2", y="pca15", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca02_v_pca15.png", dpi=500)
+print(g)
+
+
+# This graph depicts diversity within the neuroscience field as some papers are about facial recognition (negative) and other papers are about biochemistry (positive).
+
+# In[30]:
+
+
+(
+    pca_df
+    [["pca2", "pca15", "category", "document","doi"]]
+    .sort_values(["pca15", "pca2"], ascending=[False, False])
+    .head(10)
+)
+
+
+# In[31]:
+
+
+(
+    pca_df
+    [["pca2", "pca15", "category", "document","doi"]]
+    .sort_values(["pca15", "pca2"], ascending=[False, False])
+    .tail(10)
+)
+
+
+# These papers confirm that the negative axis of PCA15 is facial recognition.
+
+# ## PCA1 vs PCA8
+
+# In[32]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_02_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_08_cossim_word_cloud.png'
+)
+
+
+# The wordcloud on the right seems to represent the following concept:  biochemistry vs developmental biology. Main evidence for this appears in the plot below.
+
+# In[33]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca2", y="pca8", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca02_v_pca08.png", dpi=500)
+print(g)
+
+
+# In[34]:
+
+
+(
+    pca_df
+    [["pca2", "pca8", "category", "doi"]]
+    .query("pca2 > -2 & pca2 < 2 & pca8 < -1")
+    .category
+    .value_counts()
+    .head(10)
+)
+
+
+# In[35]:
+
+
+(
+    pca_df
+    [["pca2", "pca8", "category", "doi"]]
+    .query("pca2 > -2 & pca2 < 2 & pca8 > 1")
+    .category
+    .value_counts()
+    .head(10)
+)
+
+
+# Looking at the top left and bottom left quadrants the top categories are: biochemistry and developmental biology. Based on this confirmation I'd argue that pca8 covers both of these concepts.
+
+# ## PCA3 VS PCA13
+
+# In[36]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_02_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_13_cossim_word_cloud.png'
+)
+
+
+# Based on a quick google search the wordcloud on the right represents: viruses (immunology) vs model organisms.
```

Comment:
More specifically than model organisms, PC13 looks to be specifically reproductive biology. I'm amazed the embedding managed to place the reproductive structures in plants, worms, and flies all along the same PC.